src/main/java/mate/academy/service/UserService.java

```diff
@@ -14,7 +14,14 @@ public class UserService {
      * @return - user if his email is equal to passed email.
      * Return <code>null</code> if there is no suitable user
      */
-    public User findByEmail(String email) {
-        return null;
+    public static User findByEmail(String email) {
```

Comment:
It's better **not** to add `static` keyword to this method without a reason. What about creating removing static and regering to the UserService through the instance?